packages/in_app_purchase/in_app_purchase/README.md

```diff
@@ -245,6 +246,63 @@ InAppPurchase.instance
     .buyNonConsumable(purchaseParam: purchaseParam);
 ```
 
+### Accessing platform specific product or purchase properties
+
+The function `_inAppPurchase.queryProductDetails(productIds);` will give you a `ProductDetailsResponse` with a list of 
+purchasable products of type `List<ProductDetails>`. This `ProductDetails` class is a platform independent class 
+with only properties available on both platforms. However, when you want to access some specific
+platform properties this is also possible. The `ProductDetails` object is of subtype `GooglePlayProductDetails`
+when the platform is Android and `AppStoreProductDetails` on iOS. Accessing the skuDetails, resp. skProduct gives
+you all the information that is available in the original platform objects.
+
+This is an example on how to get the `introductoryPricePeriod` on Android:
+```dart
+import 'package:in_app_purchase_android/in_app_purchase_android.dart';
+
+if (productDetails is GooglePlayProductDetails) {
+  var skuDetails = (productDetails as GooglePlayProductDetails).skuDetails;
+  print(skuDetails.introductoryPricePeriod);
+}
+```
+
+And this way you get the subscriptionGroupIdentifier of a subscription on iOS:
+```dart
+import 'package:in_app_purchase_ios/in_app_purchase_ios.dart';
+
+if (productDetails is AppStoreProductDetails) {
+  var skProduct = (productDetails as AppStoreProductDetails).skProduct;
+  print(skProduct.subscriptionGroupIdentifier);
+}
+```
+
+The `purchaseStream` gives you objects of type `PurchaseDetails`. This object gives you all information that is 
+available on any platform, such as purchaseID and transactionDate, but here you can also access the platform 
```

Comment:
I learned from Stuart that we should try to refrain from using "you" as it might lead to confusion (who is the "you" that is being referred to).

I also suggest to changed "on any platform" into "on all endorsed platforms", this will limit the scope to all platforms that are guaranteed to be supported by this plugin: 

```suggestion
available on all endorsed platforms, such as purchaseID and transactionDate, but here it is also possible to access the platform 
```